tasks/e2e.sh

```diff
@@ -135,7 +138,7 @@ npm install $cli_path
 
 # Install the app in a temporary location
 cd $temp_app_path
-create_react_app --scripts-version=$scripts_path test-app
+create_react_app --scripts-version=$scripts_path --template=$root_path/packages/react-scripts/fixtures/kitchensink test-app
```

Comment:
I've been doing some tests, and the slowest part of the travis run is installing node modules.
By splitting the test suite, the number of `npm install` effectively doubles, so does the execution time.

Are we sure we want this?